package.json

```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "start": "node ./bin/www",
     "compile": "node_modules/.bin/gulp css",
-    "pretest": "node_modules/.bin/npm-shrinkwrap-check",
+    "lint": "eslint .",
+    "pretest": "npm-shrinkwrap-check; eslint .",
```

Comment:
I think it needs to be `npm-shrinkwrap-check && eslint .` if we want `npm-shrinkwrap-check` failures to prevent test runs